app/views/items/index.html.erb

```diff
@@ -4,6 +4,21 @@
 <% else %>
   <h1>All Items</h1>
 <% end %>
+
+<section id="most-popular-items">
+  <h3>Most Popular Items: </h3>
+  <% @items.five_most_popular_items.each do |item| %>
+    <p><%= item.name %>: <%= item.purchase_amount %> purchased</p>
+  <% end %>
+</section>
+
+<section id="least-popular-items">
+  <h3>Least Popular Items: </h3>
+  <% @items.five_least_popular_items.each do |item| %>
+    <p><%= item.name %>: <%= item.purchase_amount %> purchased</p>
+  <% end %>
+</section>
```

Comment:
I currently have each section calling the different methods outlined above, but this might be refactored to use a single method if we change them above.